README.md

```diff
@@ -12,38 +12,38 @@ follow the specfication.
 ### Project status
```

Comment:
README.md has been separately updated in https://github.com/ForensicArtifacts/artifacts/pull/346